reenact/templates/reenact/map.html

```diff
@@ -0,0 +1,31 @@
+{% extends 'base.html' %}
+
+{% block title %}
+  Karte
+{% endblock title %}
+{% block sources %}
+  <link rel="stylesheet"
+        href="https://unpkg.com/leaflet@latest/dist/leaflet.css" />
+  <script src="https://unpkg.com/leaflet@latest/dist/leaflet.js"></script>
```

Comment:
These should be moved to `css` and `javascript` block each.
In combination with {{block.super}} in order to inherit tailwind.css, project.css